embassy-stm32/src/rcc/bd.rs

```diff
@@ -18,10 +25,20 @@ impl From<LseDrive> for crate::pac::rcc::vals::Lsedrv {
         match value {
             #[cfg(any(rtc_v2f7, rtc_v2l4))]
             LseDrive::Low => Lsedrv::LOW,
+            #[cfg(any(rtc_v2h7))]
+            LseDrive::Low => Lsedrv::LOWEST,
+            #[cfg(pwr_h7rm0433)] // Errata ES0392 2.2.14
+            LseDrive::MediumLow => Lsedrv::MEDIUMHIGH,
+            #[cfg(not(pwr_h7rm0433))]
             LseDrive::MediumLow => Lsedrv::MEDIUMLOW,
+            #[cfg(pwr_h7rm0433)]
+            LseDrive::MediumHigh => Lsedrv::MEDIUMLOW,
+            #[cfg(not(pwr_h7rm0433))]
```

Comment:
I'm not sure if this is the right place to fix this, or if it should be the register definitions that change.

Reply:
Is mediumlow supported across all chips but mediumhigh isn't? If so, perhaps mediumlow should be the default.

Reply:
The RM0433 chips have an errata where the bits for MEDIUMLOW actually set the drive level to Medium High and vise versa.

Reply:
And if I'm going to update the register definitions I may as well rename the H7's `LOWEST` -> `LOW` to match the other families.

Reply:
I think it'd be best to fix this in the registers, so I made a PR there: https://github.com/embassy-rs/stm32-data/pull/272